sklearn/ensemble/base.py

```diff
@@ -84,6 +84,8 @@ class BaseEnsemble(six.with_metaclass(ABCMeta, BaseEstimator,
     estimators_ : list of estimators
         The collection of fitted base estimators.
     """
+    # overwrite _required_parameters from MetaEstimatorMixin
```

Comment:
Add a comment as a reminder that this is because all inheriting classes are default constructible, and that if you write an inheriting class that isn't, make sure to override the attribute?